hpx/util/unwrapped.hpp

```diff
@@ -495,20 +507,66 @@ namespace hpx { namespace util
                     util::detail::pack<typename std::decay<Ts>::type...>
                 >::type type;
         };
-    }
-
-    ///////////////////////////////////////////////////////////////////////////
+    } // end namespace detail
```

Comment:
As it seems doxygen hides namespaces named as `detail` by default. Shall I still add the comments then?